docs/knowledge_base/notifications.md

```diff
@@ -1,14 +1,10 @@
----
-title: Notifications
-category: 6255e67693d5200013b1fa41
-hidden: false
----
+# Notifications
 
-# Overview
+## Overview
 
```

Comment:
`Overview` sections are silly... That's just what the top of the page is. You don't have to say it out loud.

```suggestion
```